script/Arcueid/Day 10/10_00_ARC10_3_3.txt

```diff
@@ -18,7 +18,7 @@ We enter the park.
 -- 平日の午前中でも利用者は多かった。背広姿の社会人、子供連れの女性、俺よりやや年上のペアなどが、のんびりと過ごしている。
 Even on a weekday morning, the park is teeming with people.
  Businessmen in suits, women and their children, a handful of slightly older couples...
- All sorts of people are out here leisurely enjoying the day.
+ all sorts of people are out here leisurely enjoying the day.
```

Comment:
Feels like 2 distinct sentences here.

Reply:
So put "and" before "a handful of slightly older couples"?

Reply:
No, just revert. "All sorts of people are out here leisurely enjoying the day" is a fine sentence.